pocs/camera/libfli.py

```diff
@@ -62,22 +62,35 @@ def __init__(self, library_path=None, *args, **kwargs):
                 locate the library.
             OSError: raises if the ctypes.CDLL loader cannot load the library.
         """
-        super().__init__(*args, **kwargs)
-        self._CDLL = load_library(name='fli', path=library_path, logger=self.logger)
+        super().__init__(name='fli', library_path=library_path, **kwargs)
+
+    # Public methods
 
+    def get_SDK_version(self):
         # Get library version.
         version = ctypes.create_string_buffer(64)
         length = ctypes.c_size_t(64)
         self._call_function('getting library version', self._CDLL.FLIGetLibVersion, version, length)
-        self._version = version.value.decode('ascii')
+        return version.value.decode('ascii')
 
-    # Properties
+    def get_cameras(self):
+        """Convenience function to get a dictionary of all currently connected camera serial numbers
+        and their corresponding device nodes.
+        """
+        device_list = self.FLIList(interface_type=c.FLIDOMAIN_USB,
+                                   device_type=c.FLIDEVICE_CAMERA)
+        if not device_list:
+            raise error.PanError("No FLI camera devices found.")
 
-    @property
-    def version(self):
-        return self._version
+        cameras = {}
+        for device in device_list:
+            port = device[0]
+            handle = self.FLIOpen(port)
```

Comment:
Good point. Should wrap this in `try...except` in case a camera somehow drops out between getting the list of connected cameras and opening them each to get their serial numbers.